src/IQFeed.CSharpApiClient/Streaming/Level2/Messages/UpdateSummaryMessage.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Globalization;
+using IQFeed.CSharpApiClient.Extensions;
+
+namespace IQFeed.CSharpApiClient.Streaming.Level2.Messages
+{
+    public class UpdateSummaryMessage
+    {
+        public const string UpdateMessageTimeFormat = "HH:mm:ss.ffffff";
+        public const string UpdateMessageDateFormat = "yyyy-MM-dd";
+
+        public UpdateSummaryMessage(string symbol,
+                                    string mmID,
+                                    float bid,
+                                    float ask,
+                                    int bidSize,
+                                    int askSize,
+                                    DateTime bidTime,
+                                    DateTime date,
+                                    string conditionCode,
+                                    DateTime askTime,
+                                    string bidInfoValid,
+                                    string askInfoValid,
+                                    string endOfMessageGroup)
+        {
+            Symbol = symbol;
+            MMID = mmID;
```

Comment:
do you prefer to stick with MMID with all the IDE complains about it or rename to MarketMakerId